app/src/main/java/com/openclassrooms/realestatemanager/presentation/utils/customView/AddPropertyInformationCustomView.java

```diff
@@ -0,0 +1,39 @@
+package com.openclassrooms.realestatemanager.presentation.utils.customView;
+
+import android.content.Context;
+import android.util.AttributeSet;
+import android.view.LayoutInflater;
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.constraintlayout.widget.ConstraintLayout;
+
+import com.openclassrooms.realestatemanager.R;
+
+public class AddPropertyInformationCustomView extends ConstraintLayout {
+
+    public AddPropertyInformationCustomView(@NonNull Context context) {
+        super(context);
+    }
+
+    public AddPropertyInformationCustomView(@NonNull Context context, @Nullable AttributeSet attrs) {
+        super(context, attrs);
+    }
+
+    public AddPropertyInformationCustomView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr) {
+        super(context, attrs, defStyleAttr);
+    }
+
+    public AddPropertyInformationCustomView(@NonNull Context context, @Nullable AttributeSet attrs, int defStyleAttr, int defStyleRes) {
+        super(context, attrs, defStyleAttr, defStyleRes);
+    }
+
+    @Override
+    protected void onFinishInflate() {
+        super.onFinishInflate();
+
+        View addPropertyInformation = LayoutInflater.from(getContext()).inflate(R.layout.custom_view_add_property_information,this,false);
```

Comment:
`addView()` is not necessary. 
See [https://stackoverflow.com/questions/4328838/create-a-custom-view-by-inflating-a-layout](url)

Reply:
done